_component_design/loading-indicators.md

```diff
@@ -0,0 +1,205 @@
+![]({{site.url}}/assets/images/components/processing/loading-indicator/spinner-text-only.svg)
+
+{% endfigure %}
+{% endcolumn %}
+
+{% column right:"hxCol-4 hxCol-xs-6 hxCol-sm-6 hxCol-md-4 hxCol-lg-4" %}
+{% figure [caption:"Loading text, when used alone, should always use ellipses. By default, the text uses a type-size of 16px."] [class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/components/processing/loading-indicator/spinner-text-font.svg)
+
+{% endfigure %}
+{% endcolumn %}
+
+</div>
+
+## Animation
+
+<div class="hxRow">
+
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+Animation can be used in loading indicators to visually indicate that back-end systems are performing load operations. The following are best practice for animating spinner icons:
+
+- Use a simple animation.
+- Rotate icons 360 degrees in 0.8 seconds.
+- Always rotate icons clockwise.
+
+{% endcolumn %}
+
+{% column right:"hxCol-4 hxCol-xs-6 hxCol-sm-6 hxCol-md-4 hxCol-lg-4" %}
+{% figure [caption:"Spinner icons rotate 360 degrees in 0.8sec."] [class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/components/processing/loading-indicator/spinner-360.svg)
+
+{% endfigure %}
+{% endcolumn %}
+
+{% column right:"hxCol-4 hxCol-xs-6 hxCol-sm-6 hxCol-md-4 hxCol-lg-4" %}
+{% figure [caption:"Spinner icons must always rotate clockwise."] [class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/components/processing/loading-indicator/spinner-rotate-clockwise.svg)
+
+{% endfigure %}
+{% endcolumn %}
+
+</div>
+
+## Layout
+
+<div class="hxRow">
+
+Loading indicators are generally centered within their given component. The following are examples of loading indicators inside various components of the UI:
+
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+- Loading spinner in dashboard tiles.
+
+{% endcolumn %}
+
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+{% figure [caption:""] [class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/components/processing/loading-indicator/spinner-tiles.svg)
+
+{% endfigure %}
+{% endcolumn %}
+
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+- Loading spinner in cards.
+
+{% endcolumn %}
+
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+{% figure [caption:""] [class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/components/processing/loading-indicator/spinner-cards.svg)
+
+{% endfigure %}
+{% endcolumn %}
+
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+- Loading spinner in modals.
+
+{% endcolumn %}
+
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+{% figure [caption:""] [class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/components/processing/loading-indicator/spinner-modals.svg)
+
+{% endfigure %}
+{% endcolumn %}
+
+</div>
```

Comment:
I wonder if these three would be better served as a three-up layout with the bullet points turning into captions. See Month-picker section as an example.
![image](https://user-images.githubusercontent.com/586682/32229622-46a5c438-be1f-11e7-8584-ff85b96393f7.png)

Reply:
See the following for column values: https://raw.githubusercontent.com/rackerlabs/design-system/staging/_component_design/date-picker.md